integration/spark2/src/main/scala/org/apache/carbondata/spark/thriftserver/CarbonThriftServer.scala

```diff
@@ -34,10 +36,23 @@ object CarbonThriftServer {
     import org.apache.spark.sql.CarbonSession._
 
     val sparkConf = new SparkConf(loadDefaults = true)
+
+    val logger: Logger = LoggerFactory.getLogger(this.getClass)
+    if (args.length != 1 && args.length != 4) {
+      logger.error("parameters: storePath [access-key] [secret-key] [s3-endpoint]")
```

Comment:
After this change, how user can use this class for HDFS cluster?

Reply:
Fix it, now it support local, HDFS and S3 mode